src/haz3lweb/Log.re

```diff
@@ -36,6 +36,9 @@ let is_action_logged: Update.t => bool =
   | ResetSlide
   | ToggleMode
   | SwitchSlide(_)
+  | SwitchTextEditor
+  | UpdateTitle(_)
```

Comment:
Let's group all Exercise related actions into an ExerciseAction.t type that is injected into Action.t at once, to avoid having too many cases?